### PR TITLE
[GDR-2190] Wide table in gDRin

### PR DESCRIPTION
**Pull request by j-smola**
# Description
## What changed?
Related JIRA issue:  [GDR-2190](https://jira.gene.com/jira/browse/GDR-2190)
* add row names to dt created by `fit_FUN` in `fit_SE` and by `rbindParallelList` in `fit_SE.combination` (it is required by `gDRutils::.convert_se_assay_to_dt` for `wide_structure = TRUE`)

## Why was it changed?
* to increase readability of assay tables in gDRin

# Checklist for sustainable code base
- [x] I added tests for any code changed/added
- [x] I added documentation for any code changed/added
- [x] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [x] Package version bumped
- [x] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by gladkia — APPROVED** [review]

File: R/fit_SE.R [hunk 1]
```diff
@@ -100,17 +101,27 @@ fit_FUN <- function(x,
...
+        range_conc = range_conc,
+        force_fit = force_fit,
+        pcutoff = pcutoff,
+        cap = cap,
+        normalization_type = curve_type)
+    )
+    
+    if (is.null(fit_df$fit_source)) {
+      rownames(fit_df) <- fit[["normalization_type"]]
+    } else {
```
Comment (bczech) [inline R/fit_SE.R:120]: If these changes are required for working other PRs all the datasets should be reprocessed :/

File: R/fit_SE.R [hunk 2]
```diff
@@ -100,17 +101,27 @@ fit_FUN <- function(x,
...
+        GR_0 = 1,
+        n_point_cutoff = n_point_cutoff,
+        range_conc = range_conc,
+        force_fit = force_fit,
+        pcutoff = pcutoff,
+        cap = cap,
+        normalization_type = curve_type)
+    )
+    
+    if (is.null(fit_df$fit_source)) {
```
Comment (bczech) [inline R/fit_SE.R:118]: Is it possible? How often does it occur? This function adds this https://github.com/gdrplatform/gDRutils/blob/master/R/fit_curves.R#L142

Comment (j-smola) [reply R/fit_SE.R:118]: This is for safety reasons - if the impossible happens 😉

File: R/fit_SE.R (hunk 1 above)
Comment (j-smola) [reply R/fit_SE.R:120]: Ok, with [this fix in gDRutils](https://github.com/gdrplatform/gDRutils/pull/89/files#diff-769715cb0fa3d93b007073d337903aaf1348828ffecf761270e5d58073942982R125-R127), this PR will be closed. So, reprocessing of all the datasets will not be necessary.

**Comment by j-smola** [comment]
Thanks to the change in gDRutils - changes to this PR are no longer needed.

**Pull request closed without merging**